tests/integration/integration_test.go

```diff
@@ -183,11 +207,28 @@ var _ = Describe("PCTL", func() {
 				Expect(err).ToNot(HaveOccurred())
 				return strings.Split(string(session), "\n")
 			}
+
 			Eventually(listCmd).Should(ContainElements(
-				"NAMESPACE	NAME  	SOURCE         ",
-				"default  \tmy-sub\tfoo/bar/v0.1.0\t",
+				"NAMESPACE\tNAME                       \tSOURCE                               \tAVAILABLE UPDATES ",
+				"default  \tlong-name-to-ensure-padding\tnginx-catalog/weaveworks-nginx/v0.1.0\tv0.1.1           \t",
 			))
 		})
+
+		//When("there are no available updates", func() {
+		//	It("returns the subscriptions", func() {
+		//		listCmd := func() []string {
+		//			cmd := exec.Command(binaryPath, "list")
+		//			session, err := cmd.CombinedOutput()
+		//			Expect(err).ToNot(HaveOccurred())
+		//			return strings.Split(string(session), "\n")
+		//		}
+		//
+		//		Eventually(listCmd).Should(ContainElements(
+		//			"NAMESPACE\tNAME                       \tSOURCE                                   \tAVAILABLE UPDATES ",
+		//			"default  \tlong-name-to-ensure-padding\tfoo/multi-version-weaveworks-nginx/v0.1.0\tv0.1.1           \t",
+		//		))
+		//	})
+		//})
```

Comment:
:eyes:

Reply:
Ah! Thanks . :D I wanted to finish this test. But I ran out of time yesterday. :) I also want to add a case in which there is no new version. :)